### ISSUE TITLE
limited TO

### ISSUE THREAD

**Issue by nschneid** [carmls/snacs-guidelines#15]
My interest in the paper is limited TO the part about prepositions.

Due to my injury, I am constrained TO working from home.

Theme~>Goal?

[Reddit task_008:470]

**Comment by nschneid** [comment]
stick TO friendships instead [Reddit task_000:1145]

**Comment by nschneid** [comment]
keep TO something
vs. keep FROM doing something

**Comment by svivek** [comment]
On verbnet: https://uvi.colorado.edu/verbnet/limit-76

**Comment by nschneid** [comment]
limited TO X = unable to do anything other than X

confined TO a room

**Comment by nschneid** [comment]
Maybe it evolved from something like "confined TO prison" which entails sending.

**Comment by nschneid** [comment]
Theme~>Goal

**Comment by esmanning** [comment]
Related but not quite the same:

There are some limitations TO what the parents can decide, like things that hurt the children.

Theme?

**Comment by nschneid** [comment]
Agreed that they are not quite the same. In fact, FrameNet has a [Limiting](https://framenet2.icsi.berkeley.edu/fnReports/data/frame/Limiting.xml) frame in a causative relation to a [Limitation](https://framenet2.icsi.berkeley.edu/fnReports/data/frame/Limitation.xml) frame. Limitation has just an Entity as the core role, whereas Limiting involves a Range_of_options and a Characteristic.

So why is TO used with "limitation"? Maybe it was carried over from "limit" even though the meaning is slightly different?

Searching the web, it seems like "limitation(s) to the" occurs mostly in high register discourse (legalese, scientific text) and chiefly with abstract and event nouns: "limitations to the study/capabilities/use of.../power of...". The simplest example I could find: "There are limitations to the technology." One reading is that it ascribes limitations as a property of the technology, i.e., Gestalt~>Goal (though this would be a weird combination; we do not currently have this construal in the database).

There is also limitation IN: "limitations in the current study" is a frequent alternative to "limitations to the current study". The medical sense of a physical limitation seems stronger with IN, e.g. "limitations in the motion of multiple joints".
